### ISSUE TITLE
Need new LibPack for FreeCAD 0.20

### ISSUE THREAD

**Issue by donovaly** [apeltauer/FreeCAD#26]
@apeltauer , FreeCAD 0.20 is already in feature freeze. For this release we need a new LibPack with OCC 7.6.1.

- Meanwhile, Gmsh made anew release that we want to use.
-It also turned out that the file "libclang.dll" is necessary for shiboken2.
- Then I found out that the mkl_*.dll files are missing.

For these things, here is a ZIP whose content needs to be put into the Bin folder: https://github.com/donovaly/FreeCADInstProj/releases/download/0.19.3pre/updates_for_LibPack.zip

Please also have a look at #21, #22 and #24. (#23 can be closed.)

**Comment by donovaly** [comment]
I am not allowed to place the ZIP here, so I put it into my repository:
https://github.com/donovaly/FreeCADInstProj/releases/download/0.19.3pre/updates_for_LibPack.zip

**Comment by donovaly** [comment]
I added the missing file I reported in #22 now to the ZIP file linked above.

**Comment by apeltauer** [comment]
i will try to create a new libpack  asap..

**Comment by donovaly** [comment]
Many thanks! Please tell me when and how I could help.

**Comment by nortikin** [comment]
is it python 3.9?

**Comment by apeltauer** [comment]
No, Python is still 3.8...

**Comment by donovaly** [comment]
No. This is on purpose because this would break the compatibility with Windows 7 and FreeCAD has still a large userbase using Win 7.

**Comment by donovaly** [comment]
In our FreeCAD developer meeting we created a new repository in FreeCAD: https://github.com/FreeCAD/FreeCAD-LibPack

I released your LibPack + my changes from this issue here: https://github.com/FreeCAD/FreeCAD-LibPack/releases/tag/1.0

We would be happy if you could help to create a LibPack for FreeCAD 0.20. I listed the requirements here: https://github.com/FreeCAD/FreeCAD-LibPack/blob/master/Requirements.txt

- I think we can start from this LibPack: https://github.com/FreeCAD/FreeCAD-LibPack/releases/tag/1.0
- Then we need OCC 7.6.1 but with all the fixes collected by user BlobFish
- we need Qt 5.15.3

And that is it for now. After FreeCAD 0.20 was released we have time to work to get a newer Boost, Python 3.8.x etc.

Could you please have a look if the work done by @scottmudge regarding OCC and Netgen help you?: https://github.com/FreeCAD/FreeCAD-LibPack/releases/tag/2.0

**Comment by apeltauer** [comment]
@donovaly  Can you please find out what files are fixed and modified for the fix? I cannot find out what the difference are between the patch and the current version. Maybe Chennes can provide the names of the files. I will then patch those changes in the current occt 7.6.1 version....

**Comment by donovaly** [comment]
> Can you please find out what files are fixed and modified for the fix?

Do you mean the changes for OCC 7.6.1 in the https://github.com/FreeCAD/FreeCAD-LibPack/releases/tag/2.0 ?

In this case please open an issue there: https://github.com/FreeCAD/FreeCAD-LibPack/issues and ask @scottmudge .

**Comment by donovaly** [comment]
Hi @apeltauer, we made progress and have now a LibPack for FreeCAD 0.20:
https://github.com/FreeCAD/FreeCAD-LibPack/releases/tag/2.4

It was 10 days hard work. For Qt we had to surrender, however for OCC and Netgen we succeeded. I described the build process of OCC and Netgen here:
https://wiki.freecadweb.org/Compile_on_Windows#Compiling_Open_Cascade
and
https://wiki.freecadweb.org/Compile_on_Windows#Compiling_Netgen

For OCC we used @blobfish's version of OCC 7.6.2. For Netgen we purposely use the slightly older version 6.2.2105
For Python, I only update the release files to Python 3.8.10 while the debug files remain at 3.8.6 (was not worth the effort).

Can you please give it a try and report if you find issues. Should we maybe update other 3rdparty libraries?

**Comment by apeltauer** [comment]
I am on a holiday trip with no notebook.... :)
I will test it when I am back...
